packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/adjust/adjust.controller.js

```diff
@@ -0,0 +1,66 @@
+import get from 'lodash/get';
+import inRange from 'lodash/inRange';
+
+import { DELETE_CONFIRMATION_INPUT } from './adjust.constants';
+
+export default class KubernetesNodePoolsDeleteCtrl {
+  /* @ngInject */
+  constructor($translate, Kubernetes) {
+    this.$translate = $translate;
+    this.Kubernetes = Kubernetes;
+    this.DELETE_CONFIRMATION_INPUT = DELETE_CONFIRMATION_INPUT;
+  }
+
+  $onInit() {
+    const { autoscale } = this.nodePool;
+    const { lowest, desired, highest } = this.autoscaling.nodes;
+
+    this.isAjusting = false;
```

Comment:
```suggestion
    this.isAdjusting = false;
```

Reply:
Fixed by using scaling instead of ajusting.